.github/workflows/bot-pr-updated.yml

```diff
@@ -63,3 +28,13 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
       run: .github/check-wiki-changelog.sh
+  
+  spell-check:
+    name: Spell Check with Typos
```

Comment:
I think this would be needed when something in `doc/` changes ?

Reply:
Yes, I forgot to move the ignore out of this file